.env.example

```diff
@@ -0,0 +1,4 @@
+COMPOSE_PROJECT_NAME=viame_web
+PUBLIC_DATA_PATH=./girder_data
+GIRDER_ADMIN_USER=admin
+GIRDER_ADMIN_PASS=letmein
```

Comment:
Related to using `ldc` as a convenience instead of being mandatory, I think this should be moved into the `docker` directory, as that's where it's being used, and would follow the separation of project files into their respective folders.

Reply:
I _don't_ think docker compose will natively notice it if you put it in there, need to verify.

Reply:
You're right, I don't think docker will notice if it's not being run in the same directory, but I usually run `docker-compose` from the `docker` folder, because that's where all of the compose files are.